docs/src/main/sphinx/release/release-414.md

```diff
@@ -0,0 +1,52 @@
+# Release 414 (19 Apr 2023)
+
+## General
+
+* Add [recursive member access](json_descendant_member_accessor) to the
+  [JSON path language](json-path-language). ({issue}`16854`)
+* Add the [`sequence()`](built_in_table_functions) table function. ({issue}`16716`)
+* Add support for progress estimates when
+  [fault-tolerant execution](/admin/fault-tolerant-execution) is enabled. ({issue}`13072`)
+* Add support for `CUBE` and `ROLLUP` with composite sets ({issue}`16981`)
+* Add experimental support for tracing using [OpenTelemetry](https://opentelemetry.io/).
+  This can be enabled by setting the `tracing.enabled` configuration property to
+  `true` and configuring the
+  [OLTP/gRPC endpoint](https://opentelemetry.io/docs/reference/specification/protocol/otlp/)
+  by setting the `tracing.exporter.endpoint` configuration property if needed. ({issue}`16950`)
+* Fix query failure for recursive queries involving lambda expressions. ({issue}`16989`)
+* Fix incorrect results when using the {func}`sequence` function with values
```

Comment:
did you need to change this to the ref?

Reply:
No, this one's correct. The sequence table function mentioned above is the anchor, but this is a fix for the existing function.